apiary.apib

```diff
@@ -2981,6 +2981,18 @@ Get a list of project items.
                 + updated_at: `2018-12-05T16:44:33+00:00` (string)
         + pagination (Pagination)
 
+### projectItems.update [POST /projectItems.update]
+
+Update a project item.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string, required)
+        + unit_internal_cost: `123.30` (number, optional)
```

Comment:
Q: Shouldn't this be a money object? With an amount and a currency code?

Reply:
Yes, fixed.